README.md

```diff
@@ -534,22 +550,26 @@ spec:
   versions:
   - version: "*"
     containers:
-    - .spec.template.spec.containers
-    - .spec.template.spec.initContainers
-    volumes: .spec.template.spec.volumes
+    - path: .spec.template.spec.containers[*]
+      name: .name
+      env: .env
+      volumeMounts: .volumeMounts
+    - path: .spec.template.spec.initContainers[*]
+      name: .name
+      env: .env
+      volumeMounts: .volumeMounts
+    volumes: .spec/template.spec.volumes
 ```
 
-## Reconciler Implementation
-
-A reconciler implementation **MUST** support mapping to PodSpec-able resources without defining a Workload Resource Mapping for those types.  If no Workload Resource Mapping exists for the `ServiceBinding` workload resource type and the workload resource is not PodSpec-able, the reconciliation **MUST** fail.
+Note: this example is equivalent to not specifying a mapping, or specifying an empty mapping.
 
-If a `ClusterWorkloadResourceMapping` defines `containers`, the reconciler **MUST** first resolve a set of candidate locations in the workload resource addressed by the `ServiceBinding` using the `Container` type (`.envs`, `.volumeMounts`) for all available containers and then filter that collection by the `ServiceBinding` `.spec.workload.containers` filter before applying the appropriate modification.
+## Reconciler Implementation
 
-If a `ClusterWorkloadResourceMapping` defines `.envs` and `.volumeMounts`, the reconciler **MUST** first resolve a set of candidate locations in the workload resource addressed by the `ServiceBinding` for all available containers and then filter that collection by the `ServiceBinding` `.spec.workload.containers` filter before applying the appropriate modification.
+A reconciler implementation **MUST** support mapping to PodSpec-able resources without defining a `ClusterWorkloadResourceMapping` for those types.
```

Comment:
Does this mean that a reconciler MUST be able to detect if resource is PodSpec-able? Or there is a simple a list of PodSpec-able resources that are supported without adding  `ClusterWorkloadResourceMapping` ?

Reply:
Neither. It's saying that unless a ClusterWorkloadResourceMapping is define, treat the workload resource as if it was PodSpecable. If it's not PodSpecable I'd expect validation to fail at admission time.

Reply:
> unless a ClusterWorkloadResourceMapping is defined, treat the workload resource as if it was PodSpecable. If it's not PodSpecable I'd expect validation to fail at admission time

Having the above sentence in the spec text would be very helpful.

Reply:
refined the text, take a look